Update new-end-user-resources.md to mention the OTel in practice talks (#1652)
open-telemetry/opentelemetry.io: content/en/blog/2022/new-end-user-resources.md

discover best practices, and meet others who are also on a journey to implement
observability powered by OpenTelemetry.

> ### OpenTelemetry In Practice Talk Series
> 
> Welcome to the **OpenTelemetry in Practice** series! This is a new experiment by
> some OpenTelemetry contributors in the End User Working Group.

Reviewer: ```suggestion
### OpenTelemetry In Practice

**OpenTelemetry in Practice** is an experimental **series** of talks initiated
by some members of the End User Working Group.
```
